### ISSUE TITLE
Issue: PEP 541 Request: pytailor

### ISSUE THREAD

**Issue by agrav** [pypa/pypi-support#634]
**Project to be claimed**

`PROJECT_NAME`: https://pypi.org/project/pytailor

**Your PyPI username**

`USER_NAME`: https://pypi.org/user/agrav

**Reasons for the request**

I believe the project has been abandoned. It is not found when searching pypi, and the presumably corresponding github repository is archived.

**Maintenance or replacement?**

I plan to publish a different package. The code repository for the new project is: https://github.com/entailor/pytailor

**Contact and additional research**

The project "pytailor" was not found when searching on pypi, but I got an authorization error when trying to use the project name, which indicated that it is an existing project.

I found an archived github project (https://github.com/bradleygolden/pytailor) with pypi installation instructions (pip install pytailor) in the readme which indicates that an actual pypi project exists with the name "pytailor".

I have tried to contact the owner of this repository through those communication channels that was found using google, but with no response.

**Comment by jamadden** [comment]
The `pytailor` project on PyPI was deleted in 2018, and I don't see any other reason it wouldn't be available. 

You didn't include the full error you saw, but I wonder if it was the error about [filename reuse](https://pypi.org/help/#file-name-reuse)? The old pytailor project published versions 0.1.0 through 0.1.6; you'll need to choose a different version to use that project name.

**Comment by agrav** [comment]
@jamadden Thanks for your reply. That makes sense. I was trying to publish the package with a version i that range. So I'll just have to start at 0.1.7 then. I'll give it another try.

**Comment by agrav** [comment]
Ok, to summarize. I was successful in using the `pytailor` name on pypi by bumping version to 0.1.7. Thanks again for hinting to this. The authorization error I mention arise from test.pypi.org where I was doing dry-runs. I then got: H`TTP Error 403: The user 'agrav' isn't allowed to upload to project 'pytailor'`. Turns out that the old `pytailor` project still exist in test.pypi.org. So I wrongly assumed this was the case for pypi.org as well.
